### ISSUE TITLE
contentful updates should also update live site

### ISSUE THREAD

**Issue by javaporter** [javaporter/greatway#1]

**Comment by jenius** [comment]
This works. We are using it live in http://carrot.is/blogging, and it has been working great for us for months. If you use contentful with netlify, any post updates will recompile and update the live site. Would need more details on your setup to debug, but it certainly works if you have configured correctly.

**Comment by javaporter** [comment]
@jenius thanks, I'll look into it. I was watching the webinar for roots-contentful last week and the presenter had the same issue, it's probably something small in the setup I'm missing.

**Comment by jenius** [comment]
Yeah it might be. If you drop more details we might be able to help. @joshrowley is good at this stuff
